pkg/customers/customer_search_test.go

```diff
@@ -202,129 +239,129 @@ func TestGetCustomersWithVerifiedStatus(t *testing.T) {
 
-	customers, _ := scope.GetCustomers("?query=jane+doe&email=jane.doe@gmail.com")
+	customers, _ := scope.searchCustomers("?query=jane+doe&email=jane.doe@gmail.com")
 	scope.assert.Equal(1, len(customers))
 
-	customers, _ = scope.GetCustomers("?query=jane&email=jane.doe@gmail.com")
+	customers, _ = scope.searchCustomers("?query=jane&email=jane.doe@gmail.com")
 	scope.assert.Equal(1, len(customers))
 
-	customers, _ = scope.GetCustomers("?query=john+doe&email=john.doe@gmail.com")
+	customers, _ = scope.searchCustomers("?query=john+doe&email=john.doe@gmail.com")
 	scope.assert.Equal(1, len(customers))
 
-	customers, _ = scope.GetCustomers("?query=john&email=john.doe@gmail.com")
+	customers, _ = scope.searchCustomers("?query=john&email=john.doe@gmail.com")
 	scope.assert.Equal(1, len(customers))
 
-	customers, _ = scope.GetCustomers("?query=jane+doe&email=jim.doe@gmail.com")
+	customers, _ = scope.searchCustomers("?query=jane+doe&email=jim.doe@gmail.com")
 	scope.assert.Equal(0, len(customers))
 
-	customers, _ = scope.GetCustomers("?query=jane&email=jim.doe@gmail.com")
+	customers, _ = scope.searchCustomers("?query=jane&email=jim.doe@gmail.com")
 	scope.assert.Equal(0, len(customers))
 
-	customers, _ = scope.GetCustomers("?query=jim&email=jane.doe@gmail.com")
+	customers, _ = scope.searchCustomers("?query=jim&email=jane.doe@gmail.com")
 	scope.assert.Equal(0, len(customers))
 }
 
-func TestGetCustomersUsingPaging(t *testing.T) {
+func TestsearchCustomersUsingPaging(t *testing.T) {
 	scope := Setup(t)
 	_ = scope.CreateCustomers(30, client.INDIVIDUAL)
 
 	// Get first page of 10
-	customers, _ := scope.GetCustomers("?skip=0&count=10")
+	customers, _ := scope.searchCustomers("?skip=0&count=10")
 	scope.assert.Equal(10, len(customers))
 
 	// Get second page of 10
-	customers, _ = scope.GetCustomers("?skip=10&count=10")
+	customers, _ = scope.searchCustomers("?skip=10&count=10")
 	scope.assert.Equal(10, len(customers))
 
 	// Get third page of 10
-	customers, _ = scope.GetCustomers("?skip=20&count=10")
+	customers, _ = scope.searchCustomers("?skip=20&count=10")
 	scope.assert.Equal(10, len(customers))
 
 	// Should be no 4th page of 10
-	customers, _ = scope.GetCustomers("?skip=30&count=10")
+	customers, _ = scope.searchCustomers("?skip=30&count=10")
 	scope.assert.Equal(0, len(customers))
 }
 
-func TestGetCustomersUsingPagingFailure(t *testing.T) {
+func TestsearchCustomersUsingPagingFailure(t *testing.T) {
 	scope := Setup(t)
 	_ = scope.CreateCustomers(30, client.INDIVIDUAL)
 
-	customers, _ := scope.GetCustomers("?skip=123abc")
+	customers, _ := scope.searchCustomers("?skip=123abc")
 	scope.assert.Equal(0, len(customers))
 
-	customers, _ = scope.GetCustomers("?count=123abc")
+	customers, _ = scope.searchCustomers("?count=123abc")
 	scope.assert.Equal(0, len(customers))
 
-	customers, _ = scope.GetCustomers("?skip=123abc&count=123abc")
+	customers, _ = scope.searchCustomers("?skip=123abc&count=123abc")
```

Comment:
I can't find searchCustomers method :(

Reply:
Thanks for pointing this out, this screwed up with a find/replace refactor.